### PR TITLE
[1LP][RFR] Test case for BZ 1753237

### PR DESCRIPTION
**Pull request by john-dupuy**
Automate test case for https://bugzilla.redhat.com/show_bug.cgi?id=1753237

Depends on #9391 

{{ pytest: --long-running cfme/tests/automate/generic_objects/buttons/test_generic_class_custom_button.py::test_generic_object_button_edited_request }}

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit c65a3dd25103fd64b48d30078ed90cd03e836471

The *local* fixture **`button_without_group`** is used in the following files:
- **cfme/tests/automate/generic_objects/buttons/test_generic_class_custom_button.py**
    - *test_generic_object_button_edit_multiple_times*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 829f3d5d3b4837a8fce093cda7be9ce0fddd7d25

The *local* fixture **`button_without_group`** is used in the following files:
- **cfme/tests/automate/generic_objects/buttons/test_generic_class_custom_button.py**
    - *test_generic_object_button_edit_multiple_times*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 37040e1ffef691a52ee9ea74055379f0d0eeea8e

The *local* fixture **`button_without_group`** is used in the following files:
- **cfme/tests/automate/generic_objects/buttons/test_generic_class_custom_button.py**
    - *test_generic_object_button_edit_multiple_times*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

File: cfme/tests/automate/generic_objects/buttons/test_generic_class_custom_button.py [hunk 1]
```diff
@@ -250,9 +262,8 @@ def test_ansible_playbook_generic_object_button(appliance, generic_definition):
         view.cancel.click()
 
 
-@pytest.mark.manual
-@pytest.mark.meta(blockers=[BZ(1753237)], coverage=[1753237])
-def test_generic_object_button_edit_multiple_times():
+@pytest.mark.meta(blockers=[BZ(1753237)], automates=[1753237])
+def test_generic_object_button_edit_multiple_times(button_without_group):
```
Comment (sshveta) [inline cfme/tests/automate/generic_objects/buttons/test_generic_class_custom_button.py]: One question : The test doesn't look like you are editing multiple times . Request is edited and checked if it is saved or not .
Do you think test name should be `test_generic_object_button_edited_request`

Comment (john-dupuy) [reply cfme/tests/automate/generic_objects/buttons/test_generic_class_custom_button.py]: Good point @sshveta I will update it to more accurately reflect the test. Thanks!

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 037d2581c59ed048da88b33b652a8e48acf74672

The *local* fixture **`button_without_group`** is used in the following files:
- **cfme/tests/automate/generic_objects/buttons/test_generic_class_custom_button.py**
    - *test_generic_object_button_edited_request*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Pull request merged**